params/params_nathan.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 2.9864
+    
+# e.g. "anas_krystallinus"    
+species.name <- "bordomordo "

```

Comment:
There is an extra space here that could be removed.

```suggestion
species.name <- "bordomordo"
```